app/templates/notes/notes.html

```diff
@@ -0,0 +1,108 @@
+
+<!-- Scripts -->
+<!-- <script src="{{ url_for('static', path='/js/notes.js') }}"></script> -->
+{% endblock head %}
+
+{% block page_name %}Manage Notes{% endblock page_name %}
+{% block content %}
+
+<section>
+    <div class="container">
+        <div class="table-wrapper">
+            <div class="row">
+                <div class="table-title">
+                    <div class="col-sm-6">
+                        <h1 class="font font-bold py-3">Manage Notes</h1>
+                    </div>
+                    <div class="col-sm-12">
+                        <button type="button" class="btn btn-danger" data-bs-toggle="modal" data-bs-target="#dddModal">
+                            <ion-icon name="remove-circle-outline"></ion-icon>Delete
+                        </button>
+                        <a class="btn btn-success" href="/notes/add" role="button">
+                            <ion-icon name="add-circle-outline"></ion-icon>Add New
+                            Note
+                        </a>
+                    </div>
+                </div>
+                <table class="table table-hover table-striped table-bordered">
+                    <thead>
+                        <tr>
+                            <th>
+                                <span>
+                                    <input type="checkbox" id="selectAll">
+                                    <label for="selectAll"></label>
+                                </span>
+                            </th>
+                            <th>ID</th>
+                            <th>Title</th>
+                            <th>Description</th>
+                            <th>Updated on</th>
+                            <th>Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for row in data %}
+                        <tr>
+                            <td>
+                                <span class="custom-checkbox">
+                                    <input type="checkbox" id="checkbox1" name="option[]" value="1">
+                                    <label for="checkbox1"></label>
+                                </span>
+                            </td>
+                            <td>{{ row['id']}}</td>
+                            <td>{{ row['title']}}</td>
+                            <td>{{ row['description']}}</td>
+                            <td>{{ row['timestamp']}}</td>
+                            <td class="float-right">
+                                <a href="/notes/view/{{ row['id'] }}" title="Edit" class="action-icon">
+                                    <ion-icon name="pencil-outline"></ion-icon>
+                                </a>
+                                <a href="#" title="Delete" class="action-icon" data-bs-toggle="modal"
+                                    data-bs-target="#dddModal">
```

Comment:
What is the meaning of the name dddModal?

Reply:
I'll modify